GroceryStore/InventoryViewController.h

```diff
@@ -0,0 +1,13 @@
+//
+//  InventoryViewController.h
+//  GroceryStore
+//
+//  Created by Jilian Scott on 5/13/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface InventoryViewController : UIViewController
```

Comment:
For client projects, we would prefix all classes with a 3 capital letter prefix.  For this assignment, it doesn't matter though.